tests/app/views/test_drafts.py

```diff
@@ -884,8 +897,8 @@ def test_copy_draft(self):
             content_type='application/json')
 
         data = json.loads(res.get_data())
-        assert_equal(res.status_code, 201)
-        assert_equal(data['services']['lot'], 'SCS')
+        assert_equal(res.status_code, 201, res.get_data())
```

Comment:
What does the third parameter to `assert_equal` do here? I can't seem to find any documentation on it.